source/common/config/grpc_mux_subscription_impl.cc

```diff
@@ -9,7 +9,7 @@
 namespace Envoy {
 namespace Config {
 
-GrpcMuxSubscriptionImpl::GrpcMuxSubscriptionImpl(GrpcMux& grpc_mux,
+GrpcMuxSubscriptionImpl::GrpcMuxSubscriptionImpl(GrpcMuxSharedPtr grpc_mux,
                                                  SubscriptionCallbacks& callbacks,
                                                  SubscriptionStats stats,
```

Comment:
Are you sure you need a temporary `SubScriptionStats` here and copy to `stats_`?

Reply:
Because that's how the code already was, and because this class is going to be deleted soon (maybe even in this PR if that's how we decide to proceed), I think it's better to leave this alone.